modules/core/src/main/java/org/apache/ignite/configuration/SqlConfiguration.java

```diff
@@ -176,6 +179,37 @@ public SqlConfiguration setValidationEnabled(boolean validationEnabled) {
         return this;
     }
 
+    /**
+     * Sets query engines configuration.
+     * <p>
+     * There are several engines to execute SQL queries can be configured. If configured more than one engine, exact
+     * engine to execute the query can be chosen in run-time by {@code queryEngine} JDBC connection property or by
+     * {@code QUERY_ENGINE('engineName')} SQL query hint. If no query engine is explicitly chosen, default query engine
+     * will be used (see {@link QueryEngineConfiguration#setDefault(boolean)}).
+     * </p>
+     * </p>
```

Comment:
```suggestion
     * </p>
     * <p>
```
I'm not sure if closing tags are ever needed.

Reply:
Ok, I've removed the closing tag, we don't use it usually